### ISSUE TITLE
Gateways and away missions

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#10285]
Should an away mission be booted up after roundstart, you need to manually call detect and initialize on the home gateway's center piece for it to detect all of its connected pieces (So it can succesfully create a portal), and for it to actually link to the away gateway (So you can actually get to the away mission)

**Comment by Imagebard** [comment]
Yeah, that's about how I think it works.

So?

**Comment by ghost** [comment]
Well, it's kinda something that should be automated, rather than the admin having to press further buttons found out from codetrawling.

Just posting it here so a coder can see, and if an admin ain't so sure on how to get the gate working when they make an away mission mid-round.

**Comment by unid15** [comment]
The bug only appears for me when I load external .dmm files - spawning away missions through both the "Create Away Mission" and "Load Map Element" verbs works fine

**Comment by ghost** [comment]
Just loaded an away mission on the main server (IC OOC ban me) and yeah, no linky

![home Gate](http://i.imgur.com/d4hkuti.png)

**Comment by Imagebard** [comment]
I still don't quite understand. You're saying that admins need to initialize it, is that it?

**Comment by ghost** [comment]
Yes, through atom proc-calls, not something most admins know how to do and something that should , by assumption, be automated with loading an away mission.

**Comment by Imagebard** [comment]
Okay.
I'm still pretty sure the gateway worked fine, but then again there hasn't been a away mission in ages.

**Comment by unid15** [comment]
Gateways don't use these variables anymore

You have to click on the center piece as a human and the portal will appear

EDIT: nvm I was thinking of another thing

**Comment by unid15** [comment]
how are you loading the away mission?

**Comment by ghost** [comment]
Create away mission button in the admin tab.
